site/docs/api/sources-sinks.md

```diff
@@ -249,8 +249,8 @@ p.readFrom(S3Sources.s3(singletonList("input-bucket"), "prefix",
  .writeTo(Sinks.logger());
 ```
 
-The S3 sink works similar to the local file sink, writing a line to the
-output for each input item:
+The S3 sink works similarly to the local file sink, writing a line to
```

Comment:
"similar to" was actually better (easier to pronounce) and it is grammatically acceptable.